misk/web/tabs/loader/.gitignore

```diff
@@ -0,0 +1,15 @@
+# Logs
```

Comment:
should we just put a .gitignore at the root of the project? These are going to be repetitive

Reply:
yup, I'll clean these up